WordPressKit/WordPressKit/PluginDirectoryEntry.swift

```diff
@@ -151,27 +151,23 @@ extension PluginDirectoryEntry: Decodable {
 typealias Author = (name: String, link: URL?)
 
 private func extractAuthor(_ author: String) -> Author {
+    let name = author.replacingOccurrences(of: "<[^>]+>", with: "", options: String.CompareOptions.regularExpression, range: nil)
+    // Because the `author` field is so free-form, there's cases of it being
+    // * regular string ("Gutenberg")
+    // * URL ("https://wordpress.org/plugins/gutenberg/#reviews")
+    // * HTML link ("<a href="https://wordpress.org/plugins/gutenberg/#reviews">Gutenberg</a>"
+    // but also fun things like
+    // * malformed HTML: "<a href="">Gutenberg</a>".
+    // To save ourselves a headache of trying to support all those edge-cases when parsing out the
+    // user-facing name, let's just regex-strip everything that looks like HTML tag.
```

Comment:
Have you considered using XMLParser instead? It doesn't seem a lot more complicated and might be more robust

https://gist.github.com/koke/19daba3a81d9c08e0e50eaf25ec5c43d